examples/startup/data_transfer_runner.py

```diff
@@ -62,7 +62,7 @@ def main(
     assert user_token is not None
 
     ##########################
-    # Create a Client instance
+    # Create a client instance
```

Comment:
This ReStructuredText formatting within the script also looks odd in the rendered documentation and perhaps should be changed.